pages/post/index.tsx

```diff
@@ -74,6 +75,10 @@ export default function NewPost() {
   const pathname = usePathname();
   const savedPathname = useRef(pathname);
 
+  // New State for Posts and Threads
+  const [posts, setPosts] = useState([{ id: 1, content: '', threads: [] }]);
```

Comment:
useDraftStore is used to manage drafts and posting them. There is no need for the state in this page. See rest of the codebase how it is used